dependencies.yaml

```
@@ -266,6 +268,14 @@ dependencies:
               arch: aarch64
             packages:
               - cupy-cuda11x -f https://pip.cupy.dev/aarch64 # TODO: Verify that this works.
+  test_cpp:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.8"
+            packages:
+              - cuda-sanitizer-api=11.8.86
```

```suggestion
          - matrix:
              cuda: "11.8"
            packages:
              - cuda-sanitizer-api=11.8.86
          - matrix:
            packages:
```

By default, the `dependency-file-generator` tool requires that every matrix variation be covered in `specific` dependency lists like this one.

To override that, you can add an empty matrix option, which basically says "For all other matrix variations, use this `packages` list" (where this `packages` list happens to be empty in this case).

This is why CI is currently failing for this PR.